project-management/src/main/java/life/qbic/projectmanagement/application/rawdata/RawDataService.java

```diff
@@ -0,0 +1,174 @@
+import life.qbic.projectmanagement.application.measurement.MeasurementMetadata;
+import life.qbic.projectmanagement.application.sample.SampleInformationService;
+import life.qbic.projectmanagement.domain.model.experiment.ExperimentId;
+import life.qbic.projectmanagement.domain.model.measurement.MeasurementCode;
+import life.qbic.projectmanagement.domain.model.measurement.NGSMeasurement;
+import life.qbic.projectmanagement.domain.model.measurement.ProteomicsMeasurement;
+import life.qbic.projectmanagement.domain.model.project.ProjectId;
+import life.qbic.projectmanagement.domain.model.sample.Sample;
+import life.qbic.projectmanagement.domain.model.sample.SampleCode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.stereotype.Service;
+
+
+/**
+ * Raw Data Service
+ * <p>
+ * Service that provides an API to manage and query raw Data information
+ */
+@Service
+public class RawDataService {
+
+  private final SampleInformationService sampleInformationService;
+  private final MeasurementLookupService measurementLookupService;
+  private final RawDataLookupService rawDataLookupService;
+
+  @Autowired
+  public RawDataService(SampleInformationService sampleInformationService,
+      MeasurementLookupService measurementLookupService,
+      RawDataLookupService rawDataLookupService) {
+    this.measurementLookupService = Objects.requireNonNull(measurementLookupService);
+    this.rawDataLookupService = Objects.requireNonNull(rawDataLookupService);
+    this.sampleInformationService = Objects.requireNonNull(sampleInformationService);
+  }
+
+  /**
+   * Checks if there is raw Data registered for the provided experimentId
+   *
+   * @param experimentId {@link ExperimentId}s of the experiment for which it should be determined
+   *                     if its contained {@link Sample} have {@link MeasurementMetadata} with
+   *                     associated raw data
+   * @return true if experiments has measurements with associated measurements, false if not
+   */
+  public boolean hasRawData(ExperimentId experimentId) {
+    var result = sampleInformationService.retrieveSamplesForExperiment(experimentId);
+    var samplesInExperiment = result.getValue().stream().map(Sample::sampleId).toList();
+    if (samplesInExperiment.isEmpty()) {
+      return false;
+    }
+    if (measurementLookupService.countMeasurementsBySampleIds(samplesInExperiment) == 0) {
+      return false;
+    }
+    var measurements = measurementLookupService.retrieveAllMeasurementsWithSampleIds(
+        samplesInExperiment);
+    return rawDataLookupService.countRawDataByMeasurementIds(measurements) != 0;
+  }
+
+  @PostAuthorize(
+      "hasPermission(#projectId, 'life.qbic.projectmanagement.domain.model.project.Project', 'READ') ")
+  public Collection<RawData> findProteomicsRawData(String filter,
```

Comment:
The idea was to lazy load the data to the specific grid (proteomics, ngs) via a service call. 
This sadly necessitates different service calls for each measurement type, otherwise there is no way to differentiate the measurement IDs for each measurement type 🤔